magic/core/src/main/java/link/magic/android/core/relayer/urlBuilder/network/CustomNodeBaseOptions.kt

```diff
@@ -10,5 +11,7 @@ internal class CustomNodeBaseOptions internal constructor(
     @SerializedName("ETH_NETWORK") @Expose val network: CustomNodeConfiguration,
     mgboxHost: String,
     locale: String,
-    bundleId: String
-) : BaseOptions(apiKey, mgboxHost, locale, bundleId)
+    bundleId: String,
+    debugEnabled: Boolean? = false,
+    isCH: Boolean? = false
```

Comment:
I think we can remove this flag https://github.com/fortmatic/mgbox/pull/60#discussion_r1137762297